src/DependencyManagement/DependencyManager.cs

```diff
@@ -17,8 +17,11 @@
 
 namespace Microsoft.Azure.Functions.PowerShellWorker.DependencyManagement
 {
+    using Microsoft.Azure.Functions.PowerShellWorker.Messaging;
     using System.Management.Automation;
     using System.Management.Automation.Language;
+    using System.Management.Automation.Runspaces;
+    using System.Threading;
```

Comment:
Please move `using Microsoft.Azure.Functions.PowerShellWorker.Messaging;` and `using System.Threading;` to the outer using directives.
The reason `using System.Management.Automation;` is put here is because that's how you can use `PowerShell` type in the following code without having to reference to it by the full type name. So it's OK to put all `System.Management.Automation.*` namespace directive here, but not the rest.